pandora-common/src/utility/cards.ts

```diff
@@ -0,0 +1,183 @@
+import { z } from 'zod';
+import type { CharacterId } from '../character/characterTypes.ts';
+
+// Define Suits and Ranks
+const suits = [
+	'\u2661', //hearts
+	'\u2662', //diamonds
+	'\u2667', //clubs
+	'\u2664', //spades
+] as const;
+const ranks = ['2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K', 'A'] as const;
+
+// Define CardGameCard class
+export class CardGameCard {
+	public readonly suit: string;
+	public readonly rank: string;
+
+	public toString() {
+		return `${this.rank}${this.suit}`;
+	}
+
+	constructor(s?: string, r?: string) {
```

Comment:
Do the arguments need to be optional? If yes, then you can simplify the code by doing:
```ts
constructor(s: string = '\u2665', r: string = '2')
```